### PR TITLE
[CustomCom] [WIP] V3 Rewrite

### PR DESCRIPTION
**Pull request by QingyuChai**
Hello.

I'll be trying to rewrite this.

### CODE REVIEW

**Comment by QingyuChai** [comment]
Features:

- It will probably be built similar to tags.py and trigger cog
- I will try to add more argument support
- If possible, maybe embed support

File: cogs/customcom/customcom.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+
+class CustomCommands:
+    """Custom commands
+    Creates commands used to display text"""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.file_path = "data/customcom/commands.json"
+        self.c_commands = dataIO.load_json(self.file_path)
```
Comment (tekulvw) [inline cogs/customcom/customcom.py]: Please use the Config object instead of dataIO in V3

**Comment by QingyuChai** [comment]
Sorry there hasn't been much activity here. I'll try to work on it more.

**Comment by rttncraft** [comment]
Copied from Cog-Board: Something that will allow for customization on both the creators side and the users side. A combination of Custom Command & Choose; I would like to be able to create a sample list for a command and allow the user to be able to pick a number of results to be given on output.
I got the idea from this site: http://pynewbs.com/5a/a/

**Comment by tekulvw** [comment]
Can I make some changes to this? Things like...not deleting alias or general lol?

**Comment by QingyuChai** [comment]
Wait a second... Did I actually do that. I'll fix that lol

**Comment by QingyuChai** [comment]
Hello! Sorry to bother. It seems that aside from putting it into .gitignore, I also accidentally _deleted_ general.py. Is there anyway to undo this? Sorry for so many problems, first time making Pull Requests :p

**Comment by tekulvw** [comment]
What's the status on this?

**Comment by QingyuChai** [comment]
Sorry for the lack of activity, still working on it. I'll try updating more frequently.

**Comment by QingyuChai** [comment]
I'll be working on this next weekend. School started again so my timetable is a bit hectic.

**Review by palmtree5 — CHANGES_REQUESTED** [review]
Probably don't need the zip file that's in here. Also should move customcom into `redbot/cogs`

**Comment by palmtree5** [comment]
Closing in favor of #1003

**Pull request closed without merging**